pymars/drg.py

```diff
@@ -416,6 +417,32 @@ def get_rates_drg(sim_array, solution_object):
         total_edge_data[ic] = ic_edge_data
     return total_edge_data
 
+def target_error_check(solution, target_species):
+    """
+    check if target species are present in graph before attempting to remove them, 
+    if a target is absent from the graph the program will exit with error message
+
+    Parameters
+    ----------
+    nx_graph : obj
+        networkx graph object of solution
+    target_species : list
+        List of target species to search from
+
+    Returns
+    -------
+    void
+
+    """
+    targetInGraph=False;
```

Comment:
Too complex.  Do it like this:

for target in target_species
    if target not in solution.species_names
        sys.exit(...)
return

Reply:
.species_names doesnt exsist, so this didnt work. I just change to check the graphs instead of solution obj